envs/env-template

```diff
@@ -46,6 +46,9 @@ EMAIL_HOST_SERVER= # hostname of SMTP service
 EMAIL_HOST_USER= # username for SMTP service
 # To output emails to console, set SMTP_EMAIL_ENABLED=False
 SMTP_EMAIL_ENABLED=True # To output emails to console, set SMTP_EMAIL_ENABLED=False
+# Password reset link expires after (in seconds)
+PASSWORD_RESET_TIMEOUT=259200
```

Comment:
Just commenting some context for Django newbies like me! I got confused how this setting was applying not just to our email but to the links themselves - it's an actual Django setting: https://docs.djangoproject.com/en/3.2/ref/settings/#std-setting-PASSWORD_RESET_TIMEOUT